Gemfile

```diff
@@ -75,6 +75,8 @@ group :development, :test do
   gem 'rspec-rails'
 end
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/219637190-caeb4999-3826-40c7-a8ed-79d2ebb1da24.png)

- [ ] It would be nice if you add these two commands in the Install Sub-Section of your readme file becuase when i tried to run rspec then it showed me above error so kindly add these commands in your readme file to make your readme more useful and descriptive :blush:

```sh
rails db:create
rails db:migrate
```